.clang-format

```diff
@@ -36,9 +36,6 @@ PenaltyBreakBeforeFirstCallParameter: '15'
 ---
 Language: 'Proto'
 ---
-Language: 'Json'
-BasedOnStyle: llvm
```

Comment:
didn't you just merge this stuff?

Reply:
I backed it out; it broke CI. I'm super frustrated with the interaction between precommit and clang-format versions.

Reply:
Can we just rally around a specific clang format version perhaps?

Reply:
I was thinking of something a bit more drastic: https://github.com/fwupd/fwupd/pull/5488